<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Employes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(199, 203, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>971</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>GESTION</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>571</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTextEdit" name="textEdit_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;type Ã©venement&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>220</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Nombre d'invitation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>70</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nbr_invi">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>220</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;localisation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="localisation">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>170</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ajouter">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>290</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modifier">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>290</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QComboBox" name="type_event">
       <property name="geometry">
        <rect>
         <x>372</x>
         <y>120</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>mariage</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>anniversaire</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tournoi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>soiree</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>remise de diplome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>fiancaille</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>260</y>
        <width>271</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Suppression</string>
      </property>
      <widget class="QLineEdit" name="id_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Entrer l'ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supprimer">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>RECHERCHE</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>172</x>
        <y>21</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>25</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher par ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher par Type</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rech_id">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>17</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rech_type">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher par localisation</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rech_localisation">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>161</y>
        <width>531</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="type_event_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>mariage</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>anniversaire</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tournoi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>soiree</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>remise de diplome</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fiancaille</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>MAP</string>
     </attribute>
     <widget class="WebAxWidget" name="WebBrowser">
      <property name="control" stdset="0">
       <string>{8856F961-340A-11D0-A96B-00C04FD705A2}</string>
      </property>
      <property name="focusPolicy" stdset="0">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="geometry" stdset="0">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>691</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>AFFICHAGE</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>631</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_triec">
      <property name="geometry">
       <rect>
        <x>782</x>
        <y>67</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Trie_croissant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_tried">
      <property name="geometry">
       <rect>
        <x>782</x>
        <y>127</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Trie_dÃ©croissant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>460</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Convertir en pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>480</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>STATISTIQUE</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>64</x>
        <y>29</y>
        <width>821</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>type d'evenement</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>notification</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_notification">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Taper une notification</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>601</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#df3d0c;&quot;&gt;Gestion d'Ã©venement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>WebAxWidget</class>
   <extends>QAxWidget</extends>
   <header>webaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
